app/views/assignments/_form.html.erb

```diff
@@ -88,14 +140,42 @@
                          size: 35,
                          required: true,
                          placeholder: t('time.format_string.placeholder') %>
-        <%= f.label :is_hidden, Assignment.human_attribute_name(:is_hidden) %>
+        <%= f.label :is_hidden, 'Default Visibility' %>
```

Comment:
In general changes to `human_attribute_name` should be done in the localization files (in this case, `models/assignments/en.yml`).

However I do not like the term "Default" for this label, so please revert the change.